services/server/server_test.go

```diff
@@ -41,9 +41,11 @@ func TestServer(t *testing.T) {
 		require.NoError(t, sqlDB.Close())
 	}()
 
+	checker := new(MockChecker)
```

Comment:
But why we mock it? It should work in devcontainer?

Reply:
The mock here is because of:
1. We have other mocks so, to keep it consistent
2. To avoid dependencies in tests.